packages/filesystem/src/browser/location/location-renderer.tsx

```diff
@@ -42,14 +48,30 @@ export class LocationListRenderer extends ReactRenderer {
     protected readonly handleLocationChanged = (e: React.ChangeEvent<HTMLSelectElement>) => this.onLocationChanged(e);
     protected doRender(): React.ReactNode {
         const location = this.service.location;
-        const locations = !!location ? location.allLocations : [];
+        const locations = (!!location ? location.allLocations : []).map(uri => ({ uri }));
+        if (locations.length > 0 && this._drives) {
```

Comment:
why not for `locations.length === 0`?

Reply:
If there are not `locations`, hence it is empty, it means, the `root` of the model is not available, you cannot select anything. In such cases, I did not want to append the drives. What do you think?

Reply:
sounds reasonable. I was just not able to read that from the condition at L51.